cmd/legacy_main_test.go

```diff
@@ -315,6 +358,9 @@ func (t *MainTest) TestForwardedEnvVars() {
 	}, {
 		inputEnvVars:             map[string]string{"GRPC_GO_LOG_VERBOSITY_LEVEL": "99", "GRPC_GO_LOG_SEVERITY_LEVEL": "INFO"},
 		expectedForwardedEnvVars: []string{"GRPC_GO_LOG_VERBOSITY_LEVEL=99", "GRPC_GO_LOG_SEVERITY_LEVEL=INFO"},
+	}, {
+		inputEnvVars:             map[string]string{"GCSFUSE_IN_BACKGROUND_MODE": "true", "GCSFUSE_MOUNT_INSTANCE_ID": logger.MountInstanceID()},
```

Comment:
Let's simplify this by using an integer literal instead of dependency on the call or output of `logger.MountInstanceID()`.


```suggestion
		inputEnvVars:             map[string]string{"GCSFUSE_IN_BACKGROUND_MODE": "true", "GCSFUSE_MOUNT_INSTANCE_ID": <some-integer-literal>},
		expectedForwardedEnvVars: []string{"GCSFUSE_IN_BACKGROUND_MODE=true", "GCSFUSE_MOUNT_INSTANCE_ID=" + <same-integer-literal>},
```

Reply:
This won't work as the forwardedEnvVars forwards the current logger.MountInstanceID() and we can't mock that implementation without increase more complexity. So if we were to do this we need to be able to control what logger.MountInstanceID() returns. This would increase more complexity for this test.

Reply:
Then there is no point in passing GCSFUSE_MOUNT_INSTANCE_ID in the input env vars, right? If so, let's remove it from inputEnvVars because otherwise it's misleading.